zerver/lib/actions.py

```diff
@@ -2664,7 +2666,8 @@ def get_realm_user_dicts(user_profile):
     return [{'email'     : userdict['email'],
              'is_admin'  : userdict['is_realm_admin'],
              'is_bot'    : userdict['is_bot'],
-             'full_name' : userdict['full_name']}
+             'full_name' : userdict['full_name'],
+             'id'        : userdict['id']}
```

Comment:
`stream_data.neversubbed_streams_map` stores streams subscriber's id. If any user is removed from never subscribed stream, his user id needs to be removed from stream's subscriber list. See https://github.com/zulip/zulip/pull/525/commits/de8563b9fc1cbe544dc89a9fa66093e4193a15de#diff-e0b749518159d9d7dd6c9e672d1e0197R157